cwms-data-api/src/main/java/cwms/cda/data/dao/texttimeseries/RegularTimeSeriesTextDao.java

```diff
@@ -0,0 +1,307 @@
+            storeRow(connection, officeId, tsId, regularTextTimeSeriesRow, maxVersion, replaceAll);
+        });
+    }
+
+    private void storeRow(Connection connection, String officeId, String tsId,
+                          RegularTextTimeSeriesRow regularTextTimeSeriesRow,
+                          boolean maxVersion, boolean replaceAll) throws SQLException {
+
+        TimeZone timeZone = OracleTypeMap.GMT_TIME_ZONE;
+
+        String textId = regularTextTimeSeriesRow.getTextId();
+        String textValue = regularTextTimeSeriesRow.getTextValue();
+        Date dateTime = regularTextTimeSeriesRow.getDateTime();
+        Date versionDate = regularTextTimeSeriesRow.getVersionDate();
+        Long attribute = regularTextTimeSeriesRow.getAttribute();
+
+        NavigableSet<Date> dates = new TreeSet<>();
+        dates.add(dateTime);
+
+        /* the pl/sql has:
+            procedure store_ts_text(
+                  p_tsid         in varchar2,
+                  p_text         in clob,
+                  p_start_time   in date,
+                  p_end_time     in date default null,
+                  p_version_date in date default null,
+                  p_time_zone    in varchar2 default null,
+                  p_max_version  in varchar2 default 'T',
+                  p_existing     in varchar2 default 'T',
+                  p_non_existing in varchar2 default 'F',
+                  p_replace_all  in varchar2 default 'F',
+                  p_attribute    in number default null,
+                  p_office_id    in varchar2 default null)
+
+             Jooq names this one:   call_STORE_TS_TEXT - takes a date range. not used here.
+
+            and also:
+                 procedure store_ts_text(
+                      p_tsid         in varchar2,
+                      p_text         in clob,
+                      p_times        in date_table_type,
+                      p_version_date in date default null,
+                      p_time_zone    in varchar2 default null,
+                      p_max_version  in varchar2 default 'T',
+                      p_replace_all  in varchar2 default 'F',
+                      p_attribute    in number default null,
+                      p_office_id    in varchar2 default null)
+
+            Jooq names this one:   call_STORE_TS_TEXT__2  - this is what we use
+         */
+
+        if(textId != null && textValue != null){
+            // There are two storeTs methods.  You either:
+            // 1.  store a textValue at specific times but you don't care about what the textId is.
+            // 2.  make an existing textId apply at the specified times - you don't care about the current textId to textValue.
+            // This branch is if the user is trying to specify the text_Id and the text_value.
+            // We'll have to make some choices to implement this.
+            throw new UnsupportedOperationException(String.format("TextId:\"%s\" and TextValue:\"%s\" are both specified.  "
```

Comment:
This winds up returning error code 500 when I think we want a 400 here

Reply:
Yep. Not that "unsupportedOperation" is linguistically wrong.